definition.go

```diff
@@ -368,12 +369,15 @@ type IsTypeOfFn func(p IsTypeOfParams) bool
 
 type InterfacesThunk func() []*Interface
 
+type SubscribeFn func(p ResolveParams) error
```

Comment:
this is the same function signature as a FieldResolveFn, any reason not to just use that?

Reply:
Wouldn't it be more user friendly to assign this under a name which includes "Subscription" in it? Like, if you wanna search for the particular subscription handler for subscription, its easy to identify it. Could be wrong.